clinica/pydra/t1_volume/utils.py

```diff
@@ -0,0 +1,72 @@
+from os import PathLike
+from typing import Tuple, Union
+
+
+def initialize_tissues_spm_segment(
+    tissue_probability_map: Union[PathLike, None] = None
+) -> tuple:
+
+    """Prepare data structure for SPM segment interface
+
+    Parameters
+    ----------
+    tissue_probability_map: PathLike, optional
+        Path to the nifti tissue probability map. Default=None.
+
+    Returns
+    -------
+    tuple
+        "tissues" data structure for SPMSegment
+    """
+
+    from clinica.pipelines.t1_volume_tissue_segmentation.t1_volume_tissue_segmentation_utils import (
+        get_tissue_tuples,
+    )
+    from clinica.utils.spm import get_tpm
+
+    parameters = {}
+    parameters.setdefault("tissue_classes", [1, 2, 3])
```

Comment:
Why don't you set the values directly since you know the dictionary is empty? i.e:

```python
parameters["tissue_classes"] = [1, 2, 3]
...
```